syntax = "proto3";
import "google/protobuf/any.proto";
import "google/protobuf/struct.proto";
import "google/protobuf/timestamp.proto";


service EventManagerInformer {
  rpc NewEvent (stream NewEventRequest) returns (stream NewEventResponse) {}
}

message NewEventRequest {
  string type = 1;
  string instance_name = 2;
  string data = 3;
}

message NewEventResponse {
  bool is_success = 1;
  string message = 2;
}